week-3-practise/exercise-3.js

```diff
@@ -0,0 +1,14 @@
+/* In Turkish Language, some of the words are joined words and 
+they are written together. Create a new function in order to get a 
+new array with the joined words of each items in the array*/
+
+/*var newWords = [
+  { firstword: "Ata", lastword: "turk" },
+  { firstword: "Galata", lastword: "saray" },
+  { firstword: "Eski", lastword: "sehir" }
+]; */
+
```

Comment:
Have you tried implementing this one?

If we break it down, we can think about it as **transforming** each element in the array from an object into a string. Do you remember what we talked about in class about the `.map()` method? If we are transforming each element in the array into something else that is when we want to use the `.map()` method.

I'll give you another hint about working with objects. We need to use the dot (`.`), like with methods to get a *property* of the object. So in this case to join the first and last words together we could write something like this:

```js
var joinedWord = word.firstword + word.lastword;
```